### PR TITLE
Fixed sharing on Pinterest

### LINKED ISSUE #590: Share on Pinterest is broken
## Describe the bug
"Sorry, this image is broken. Please pick a different image" is the message shown when trying to share an image on Pinterest from CC Search.

## To Reproduce
1. Go to https://search.creativecommons.org/photos/7e5b0a9e-47c9-4a0e-829e-915542010c19
2. Under the share tab, click on the pinterest icon.

## Screenshots
Screenshot from 2020-02-12 23-10-36
Screenshot from 2020-02-12 23-10-40 (1)

## Desktop (please complete the following information)
- OS: Ubuntu 19.10
- Browser: chrome Version 79.0.3945.130

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
## Fixes

Fixes #590  by @makkoncept

## Description
I just changed the image url from `image.foreign_landing_url` to `image.url` with the help vue developer tool and it works fine now!

url

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ImageSocialShare.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default {
   }),
   computed: {
     imageURL() {
-      return this.image.foreign_landing_url;
+      return this.image.url;
```
Comment (brenoferreira) [inline src/components/ImageSocialShare.vue:27]: This `image.url` points to the .jpg of the image and the `foreign_landing_url` points to the image page on the source (which is what we need to share in this case). I think you need to find a different solution, maybe expose it via a different property

Comment (abhisheknaiidu) [reply src/components/ImageSocialShare.vue:31]: Done @brenoferreira I added a new `prop` then

**Comment by abhisheknaiidu** [comment]
@brenoferreira I can, but I'm not able to fix as it says `These conflicts are too complex to resolve in the editor`

**Comment by abhisheknaiidu** [comment]
@brenoferreira I removed the `merge conflicts`, Could you once review now?

File: src/router/client.js [hunk 2]
```diff
@@ -20,10 +20,6 @@ const router = new VueRouter({
       path: '/search',
       name: 'browse-page',
       component: BrowsePage,
-      // a meta field
-      meta: {
```
Comment (brenoferreira) [inline src/router/client.js:24]: can you undo this please?

File: src/router/server.js [hunk 3]
```diff
@@ -20,10 +20,6 @@ const router = new VueRouter({
       path: '/search',
       name: 'browse-page',
       component: BrowsePage,
-      // a meta field
```
Comment (brenoferreira) [inline src/router/server.js:23]: same here

**Comment by abhisheknaiidu** [comment]
@brenoferreira Yea I have done now

**Pull request merged**